config/crd/bases/cyndi.cloud.redhat.com_cyndipipelines.yaml

```diff
@@ -1,38 +1,13 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Same sort of question here. Seems like you get this from the controller-gen version.